synapse/handlers/auth.py

```diff
@@ -35,6 +36,7 @@
 
 
 class AuthHandler(BaseHandler):
+    SESSION_EXPIRE_SECS = 48 * 60 * 60
```

Comment:
We generally try and consistently always use milliseconds.

(You get brownie points for including "secs" in the name though)